src/genie/libs/parser/hp_comware/show_interface.py

```diff
@@ -0,0 +1,703 @@
+                        Optional('in_mac_pause_frames'): int,
+                        Optional('out_octets'): int,
+                        Optional('in_octets'): int,
+                        Optional('in_runts'): int,
+                        Optional('out_multicast_pkts'): int,
+                        Optional('in_frame'): int,
+                        Optional('in_broadcast_pkts'): int,
+                        Optional('out_buffer_failure'): int,
+                        Optional('out_pkts'): int,
+                        Optional('out_late_collision'): int,
+                        Optional('in_giants'): int,
+                        Optional('out_underruns'): int,
+                        Optional('in_crc_errors'): int,
+                        Optional('in_abort'): int,
+                        Optional('in_pkts'): int,
+                        Optional('last_clear'): str,
+                        },
+                    Optional('switchport'): {
+                        Optional('mode'): str,
+                        Optional('tagged'): int,
+                        Optional('untagged'): int,
+                        Optional('vlan_passing'): list,
+                        Optional('vlan_permitted'): list,
+                        Optional('encapsulation'): str,
+                    },
+                    Optional('ipv4'):
+                        {Any():
+                            {Optional('ip'): str,
+                             Optional('prefix_length'): str,
+                             Optional('secondary'): bool
+                        }
+                    },
+                }
+    }
+
+
+class DisplayInterfaces(DisplayInterfaceSchema):
+    """parser for display interface
+       display interface <interface>
+    """
+    ##########################################################
+    #      Other Plataforms equivalent command
+    ##########################################################
+    platform_equivalent_cli = ['show interfaces',
+                               'show interfaces {interface}']
+    ##########################################################
+    #      HP Comware command
+    ##########################################################
+    platform_cli = ['display interface',
+                    'display interface {interface}']
+
+    cli_command = platform_equivalent_cli + platform_cli
+
+    exclude = []
+
+    def cli(self, interface="", output=None):
+        if output is None:
+            if interface:
+                cmd = self.platform_cli[1].format(interface=interface)
+            else:
+                cmd = self.platform_cli[0]
+            out = self.device.execute(cmd)
+        else:
+            out = output
+
+        # GigabitEthernet3/8/0/38 current state: DOWN
+        p1 = re.compile(r'^ *(?P<interface>[\w\/\.\-]+) current state: (?P<enabled>[\(?\w\s\)?]+)$')
+        # IP Packet Frame Type: PKTFMT_ETHNT_2, Hardware Address: aaaa-bbbb-cccc
```

Comment:
Just to increase readability and reduce clutter, can you put a space between the regex and the following comment for the next regex? Thanks.

Reply:
Fixed